custom/icds_reports/views.py

```diff
@@ -1797,6 +1797,7 @@ def get(self, request, *args, **kwargs):
         try:
             month = int(request.GET.get('month'))
             year = int(request.GET.get('year'))
+            stream = True if request.GET.get('stream') == 'true' else False
```

Comment:
Do we need to inform the users about this option?

Reply:
You mean the file size? I believe if they know about it, they could already use that. Streaming should make things easier for them. Just a thought.

Reply:
Depending on they wrote their handler, they might have to do some extra work to handle streaming response, which is why I don't want to inform this option right away without it actually being an issue.